openwisp_radius/api/views.py

```diff
@@ -710,6 +711,9 @@ def get_serializer_context(self):
         token = default_token_generator.make_token(user)
         password_reset_urls = app_settings.PASSWORD_RESET_URLS
         default_url = password_reset_urls.get('default')
+        domain = get_current_site(self.request).domain
+        if 'example.com' not in domain:
+            default_url = default_url.replace('http://localhost:8080', domain)
```

Comment:
don't do this, is really ugly and confusing. Pass `site=site.domain` to the `format` call below.